node/commons/compatibility-layer.ts

```diff
@@ -161,6 +161,7 @@ const fillSearchItemWithSimulation = (searchItem: SearchItem, orderFormItems: Or
       seller.commertialOffer.Price = orderFormItem.sellingPrice ? orderFormItem.sellingPrice / 100 : orderFormItem.price / 100
       seller.commertialOffer.PriceValidUntil = orderFormItem.priceValidUntil
       seller.commertialOffer.ListPrice = orderFormItem.listPrice / 100
+      seller.commertialOffer.PriceWithoutDiscount = orderFormItem.price / 100
```

Comment:
i think you should also update [the line 249](https://github.com/vtex-apps/search-resolver/blob/v1.x/node/commons/compatibility-layer.ts#L249) to `oldPrice`

Reply:
you're right! Done!